osu.Game.Rulesets.Osu/Difficulty/Skills/Speed.cs

```diff
@@ -20,20 +23,116 @@ public class Speed : Skill
         private const double pi_over_4 = Math.PI / 4;
         private const double pi_over_2 = Math.PI / 2;
 
+        private const double streamaimconst = 2.42;
+
+        //  private const double stdevconst = 0.149820;
+
+        private int sdsplitcounter = 0;
+        private int sdsplitcounter2 = 0;
+        // private int runcheck1 = 0;
+
+        private double sdstrainmult = 0;
+        // private double sdstrainmult2;
+
+        // private double sdstrainmult2;
+
+        public readonly List<double> JumpDistances = new List<double>();
+        public readonly List<double> StrainTimes = new List<double>();
+        public List<double> JumpDistances2 = new List<double>();
+
         protected override double SkillMultiplier => 1400;
         protected override double StrainDecayBase => 0.3;
 
         private const double min_speed_bonus = 75; // ~200BPM
         private const double max_speed_bonus = 45; // ~330BPM
         private const double speed_balancing_factor = 40;
 
+        private double calculateStandardDeviation(IEnumerable<double> values)
+        {
+            double standardDeviation = 0;
+
+            if (values.Any())
+            {
+                // Compute the average.     
+                double avg = values.Average();
+
+                // Perform the Sum of (value-avg)_2_2.      
+                double sum = values.Sum(d => Math.Pow(d - avg, 2));
+
+                // Put it all together.      
+                standardDeviation = Math.Sqrt((sum) / (values.Count()));
+            }
```

Comment:
[This is literally still copied from stackoverflow](https://stackoverflow.com/a/3141731/6817321) (_literally_ the first result when googling the comments). Using online resources is fine but at least trim the comments since they have no value here. The mathematical concept of standard deviation is pretty widely known.

Reply:
Ok, will remove comments.

Reply:
Fixes in _e11de4b_.

Reply:
Not entirely. [I still see some commented code.](https://github.com/ppy/osu/blob/e11de4b2f373c6a85910c2d289a64d1b76f7f78a/osu.Game.Rulesets.Osu/Difficulty/Skills/Speed.cs#L90-L96)

Reply:
Fixed in 0cb6464 .